### PR TITLE
* Replace tensorflow deprecated attribute : reduction_indices -> axis

### PR DESCRIPTION
**Pull request by vfdev-5**
A minor change due to tensorflow API deprecated attribute. This PR also 'closes' [this](https://github.com/fchollet/keras/issues/4907). 

All tests [passes](https://travis-ci.org/vfdev-5/keras).

### CODE REVIEW

**Comment by fchollet** [comment]
Good catch, LGTM
